backend/src/app.ts

```diff
@@ -51,7 +51,7 @@ app.post('/user', async (req: Request, res: Response) => {
           res.send({ message: 'User exist!' });
           break;
         case ShortPasswordException:
-          res.status(400);
+          res.status(401);
```

Comment:
I think the problem here it's no change 400 to 401. The correct status code it's 400 (401 it's unauthorized), but my suggestion was about in the test expect the message to validate "it's the password is too short"!

Reply:
Done